android/pytorch_android/CMakeLists.txt

```diff
@@ -117,6 +117,8 @@ if(ANDROID_ABI)
   import_static_lib(libeigen_blas)
   import_static_lib(libcpuinfo)
   import_static_lib(libclog)
+  import_static_lib(libfmt)
```

Comment:
Hmm, isn't fmt a header only library? (And as such should not generate a static lib)?

Reply:
right, updated here and in other places